### PR TITLE
[WIP] Dev-3730 Platforms > Upgrade Quasar to latest versions

### PR DESCRIPTION
**Pull request by wilbertverayin**
## Ticket references
- [Dev-3730 Platforms > Upgrade Quasar to latest versions](https://freedom.myjetbrains.com/youtrack/issue/Dev-3730)

## Summary of changes
Add freedom components demo

## Checklist

- [ ] bugfix
- [x] new feature
- [ ] breaking change
- [ ] added unit tests
- [ ] changes introduced has been discussed and approved
- [ ] requires manual testing
- [ ] documentation has been updated

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,6 +30,9 @@
         "node-polyfill-webpack-plugin": "^1.1.4",
         "qs": "^6.5.2",
         "quasar": "^2.0.4",
+        "quasar-app-extension-f-paginate": "file:../freedom-components/f-paginate",
```
Comment (cedricabuso) [inline package.json:33]: Shouldn't we be getting this from the repo?

File: quasar.conf.js [hunk 2]
```diff
@@ -68,6 +68,10 @@ module.exports = function (ctx) {
             }
         },
         devServer: {
+            watchFiles: [
+                '../freedom-components/f-table',
```
Comment (cedricabuso) [inline quasar.conf.js:72]: Same here

File: package.json (hunk 1 above)
Comment (wilbertverayin) [reply package.json:34]: Will update how we include on package json once the freedom components are tested and merged

File: quasar.conf.js (hunk 2 above)
Comment (wilbertverayin) [reply quasar.conf.js:72]: Will update how we include on package json once the freedom components are tested and merged

**Pull request closed without merging**